keyboards/cipulot/ec_tkl/info.json

```diff
@@ -0,0 +1,434 @@
+                {"label": "2,6", "matrix": [2, 6], "x": 6.5, "y": 2.25},
+                {"label": "2,7", "matrix": [2, 7], "x": 7.5, "y": 2.25},
+                {"label": "2,8", "matrix": [2, 8], "x": 8.5, "y": 2.25},
+                {"label": "2,9", "matrix": [2, 9], "x": 9.5, "y": 2.25},
+                {"label": "2,10", "matrix": [2, 10], "x": 10.5, "y": 2.25},
+                {"label": "2,11", "matrix": [2, 11], "x": 11.5, "y": 2.25},
+                {"label": "2,12", "matrix": [2, 12], "x": 12.5, "y": 2.25},
+                {"h": 2, "label": "2,14", "matrix": [2, 14], "w": 1.25, "x": 13.75, "y": 2.25},
+                {"label": "4,14", "matrix": [4, 14], "x": 15.25, "y": 2.25},
+                {"label": "4,15", "matrix": [4, 15], "x": 16.25, "y": 2.25},
+                {"label": "3,15", "matrix": [3, 15], "x": 17.25, "y": 2.25},
+                {"label": "3,0", "matrix": [3, 0], "w": 1.75, "x": 0, "y": 3.25},
+                {"label": "3,1", "matrix": [3, 1], "x": 1.75, "y": 3.25},
+                {"label": "3,2", "matrix": [3, 2], "x": 2.75, "y": 3.25},
+                {"label": "3,3", "matrix": [3, 3], "x": 3.75, "y": 3.25},
+                {"label": "3,4", "matrix": [3, 4], "x": 4.75, "y": 3.25},
+                {"label": "3,5", "matrix": [3, 5], "x": 5.75, "y": 3.25},
+                {"label": "3,6", "matrix": [3, 6], "x": 6.75, "y": 3.25},
+                {"label": "3,7", "matrix": [3, 7], "x": 7.75, "y": 3.25},
+                {"label": "3,8", "matrix": [3, 8], "x": 8.75, "y": 3.25},
+                {"label": "3,9", "matrix": [3, 9], "x": 9.75, "y": 3.25},
+                {"label": "3,10", "matrix": [3, 10], "x": 10.75, "y": 3.25},
+                {"label": "3,11", "matrix": [3, 11], "x": 11.75, "y": 3.25},
+                {"label": "3,12", "matrix": [3, 12], "x": 12.75, "y": 3.25},
+                {"label": "4,0", "matrix": [4, 0], "w": 2.25, "x": 0, "y": 4.25},
+                {"label": "4,2", "matrix": [4, 2], "x": 2.25, "y": 4.25},
+                {"label": "4,3", "matrix": [4, 3], "x": 3.25, "y": 4.25},
+                {"label": "4,4", "matrix": [4, 4], "x": 4.25, "y": 4.25},
+                {"label": "4,5", "matrix": [4, 5], "x": 5.25, "y": 4.25},
+                {"label": "4,6", "matrix": [4, 6], "x": 6.25, "y": 4.25},
+                {"label": "4,7", "matrix": [4, 7], "x": 7.25, "y": 4.25},
+                {"label": "4,8", "matrix": [4, 8], "x": 8.25, "y": 4.25},
+                {"label": "4,9", "matrix": [4, 9], "x": 9.25, "y": 4.25},
+                {"label": "4,10", "matrix": [4, 10], "x": 10.25, "y": 4.25},
+                {"label": "4,11", "matrix": [4, 11], "x": 11.25, "y": 4.25},
+                {"label": "4,12", "matrix": [4, 12], "x": 12.25, "y": 4.25},
+                {"label": "4,13", "matrix": [4, 13], "w": 1.75, "x": 13.25, "y": 4.25},
+                {"label": "5,14", "matrix": [5, 14], "x": 16.25, "y": 4.25},
```

Comment:
JIS physical layout has a 1u-1.75u right shift cluster, same as ABNT2. this code the right shift cluster is the tsangan-esque 1.75u-1u.
Which of the two right shift clusters is intended?

Reply:
@dunk2k The board uses a 1U | 1.75U split configuration as follows:
![image](https://github.com/qmk/qmk_firmware/assets/40441626/5eb1b4f9-518e-4883-b99e-b79549740f43)
![image](https://github.com/qmk/qmk_firmware/assets/40441626/f00cbbb9-8da6-4491-894e-695e811dcdb6)

Reply:
great stuff, could highlighted lines (416 and 417) be amended to match i.e. from 1.75u-1u to 1u-175u?

Reply:
double checked, all good - my mistake 🙈